src/Compilers/VisualBasic/Portable/Analysis/FlowAnalysis/ReadWriteWalker.vb

```diff
@@ -96,6 +105,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                         Throw ExceptionUtilities.UnexpectedValue(Me._regionPlace)
                 End Select
             End If
+
+            Select Case Me._regionPlace
+                Case RegionPlace.Before, RegionPlace.After
+                    _capturedOutside.Add(variable)
```

Comment:
Probably should filter range variables here?

Reply:
I don't think VB has range variables; they act like locals

Reply:
We only add range variables if they are inside the region (for both C# and VB - see the top of this method)

Reply:
Should we do it for `_captureInside` as well?